app/views/samples/_upload_results_js.haml

```diff
@@ -57,52 +58,53 @@
+                var template = document.getElementById(`csvFileRow`).content;
+                var fragment = template.cloneNode(true);
+                fragment.querySelector(`.upload_info`).id = `upload_info_${now}`
+                fragment.querySelector(`.uploaded-samples-count`).textContent = `${samples.length} Samples ` ;
+                if (found.length != samples.length){
+                    fragment.querySelector(`.uploaded-samples-count`).innerHTML += `(<span class="dashed-underline">${(not_found.length)} sample UUID${(not_found.length>1?'s':'')}</span> not found) `
+                    fragment.querySelector(`.upload-icon`).className += ` icon-alert icon-red`
+                    fragment.querySelector(`.samples-row-action`).className += ` ttip input-required`
+                    fragment.querySelector(`.not_found_message`).id = `not_found_message_${now}`;
+                }
+                else {
+                    fragment.querySelector(`.upload-icon`).className += ` icon-checkmark`
+                }
+                fragment.querySelector(`.remove_file`).id = `remove_file_${now}`;
+                fragment.querySelector(`.file-name`).textContent = `${f.name}`;
+                
+                document.getElementById(`uploaded-files`).append(fragment)
                 
                 // if samples were not found add mouse behaviour to display tooltip
                 if (found.length != samples.length) {
-                    $(document).on("mouseover", `#not_found_mess_${evt.target.nowParam}`, function(e) {
-                        $(`#not_found_mess_${evt.target.nowParam}`)
-                            .append(`<span class="ttext" style='color:red; width: 300px;' >${not_found.join("<br>")}</span>`);
-                    })
-
-                    $(document).on("mouseleave", `#not_found_mess_${evt.target.nowParam}`, function(e) {
-                        $(`#not_found_mess_${evt.target.nowParam}`)
-                            .find(".ttext").remove();
-                    })
+                    document.getElementById(`not_found_message_${now}`).addEventListener('click', 
+                        function() {
+                            if (!document.getElementById(`ttext_${now}`)){
+                                var tooltip = document.createElement("span")
+                                tooltip.className = `ttext`
+                                tooltip.id = `ttext_${now}`
+                                tooltip.style = `color:red; width: 300px;`
+                                tooltip.innerHTML = `${not_found.slice(0,5).join("<br>")}`
+                                document.getElementById(`not_found_message_${now}`).append(tooltip)
+                                setTimeout( () => tooltip.remove() , 5000 );
+                            }
+                            else {
+                                document.getElementById(`ttext_${now}`).remove()
+                            }
```

Comment:
The problem here is that this is within the function added as click event listener and we don't have access to `fragment` there. I'll check if this can be solved, but was the first thing I tried and didn't work.

Reply:
Even if I do 

```
fragment.querySelector(".not_found_message").fragment = fragment
fragment.querySelector(".not_found_message").addEventListener('click', clickF, false )
```

and then 

```
var clickF = function(evt) {
       console.log(evt.currentTarget.fragment)
       var tooltip = evt.currentTarget.fragment.querySelector(".ttip")
       console.log(tooltip)
```

I would get a null for the query selector :S 

![image](https://user-images.githubusercontent.com/13782680/203826596-e5690144-2698-47aa-af67-57a55f46bf06.png)

Reply:
It works passing the tooltip (the result of the querySelector) as param instead of the fragment :)